java/plantuml/Portfile

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                plantuml
+version             1.2018.0
+revision            1
+categories          java editors
+platforms           darwin
+supported_archs     noarch
+license             GPL-3+
```

Comment:
This does not seem to match the download filename, which says "gplv2".

The download site provides both "GPL Java Source" and "GPL Version 2". What is the difference? Which one is supposed to be provided by this port?

Reply:
i'll try to rewrite it and use github instead.
On github it's GPL3.
https://github.com/plantuml/plantuml/blob/master/license.txt